src/api/urls.py

```diff
@@ -69,8 +71,10 @@
     path("friends/remove/", FriendRemoveListAndCreate.as_view(), name="friend-remove"),
     path("groups/", GroupList.as_view(), name="group-list"),
     path("groups/<int:pk>/", GroupDetail.as_view(), name="group-detail"),
-    path("groups/<int:pk>/add/", GroupDetailAdd.as_view(), name="group-detail-add"),
```

Comment:
if you remove these, frontend may not be able to catch up as fast so you should keep these in so that we can still be backwards compatible -- once @haiyingweng and @lucyxubroad tell you they've released a build that uses these endpoints, then you can make a PR to delete

Reply:
added the original endpoints back!